src/main/java/org/example/Main.java

```diff
@@ -4,14 +4,19 @@
 // ガターの <icon src="AllIcons.Actions.Execute"/> アイコンをクリックします。
 public class Main {
     public static void main(String[] args) {
-        //TIP ハイライトされたテキストにキャレットがある状態で <shortcut actionId="ShowIntentionActions"/> を押すと
-        // IntelliJ IDEA によるその修正案を確認できます。
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP <shortcut actionId="Debug"/> を押してコードのデバッグを開始します。<icon src="AllIcons.Debugger.Db_set_breakpoint"/> ブレークポイントを 1 つ設定しましたが、
-            // <shortcut actionId="ToggleLineBreakpoint"/> を押すといつでも他のブレークポイントを追加できます。
-            System.out.println("i = " + i);
+        // 課題5
+        // 条件式を使った課題
+        // 文字列型を使って、変数を２つ用意する
+        // その変数には苗字と名前を入れる
+        // 条件は名字の文字数が2文字以上の場合　かつ　名前の文字数が2文字以上の場合はtrue
+        // 条件がtrueの場合はその名前を表示する
+        // 条件がfalseの場合は条件に一致しませんでしたと常時する
+        String firstName = ("坂本");
+        String lastName = ("亘");
+        if( firstName.length() >= 2 && lastName.length() >= 2 ){
+            System.out.println(firstName + lastName);
+        }else{
+            System.out.println("条件に一致しませんでした。");
```

Comment:
IntelliJのactions on saveの機能で保存時自動でコードフォーマットしてほしいですね！
```suggestion
        if (firstName.length() >= 2 && lastName.length() >= 2) {
            System.out.println(firstName + lastName);
        } else {
            System.out.println("条件に一致しませんでした。");
```